CMakeLists.txt

```diff
@@ -419,6 +419,16 @@ set(SWIFT_STDLIB_MSVC_RUNTIME_LIBRARY
   ${SWIFT_STDLIB_MSVC_RUNTIME_LIBRARY_default}
   CACHE STRING "MSVC Runtime Library for the standard library")
 
+if("${CMAKE_SYSTEM_NAME}" STREQUAL "Windows" AND BOOTSTRAPPING_MODE STREQUAL "HOSTTOOLS")
```

Comment:
This can only halogen in an unsupported build, as debug builds are known to be problematic in general. Can you please also add an explicit build configuration check here?

Reply:
Sure, added a check for a debug build.